dev_guide/pod_autoscaling.adoc

```diff
@@ -68,7 +68,7 @@ For replication controllers, this scaling corresponds directly to the replicas
 of the replication controller. For deployment configurations, this will scale
 the latest deployment, or, if no deployment is present, the replication
 controller template in the deployment configuration. Note that scaling will
-occur if the latest deployment is marked as `Complete`.
+only occur if the latest deployment is marked as `Complete`.
```

Comment:
@DirectXMan12 @ironcladlou Is this more accurate? I'm not sure the last sentence is even necessary.

"For deployment configurations, this will scale the last `Complete` deployment, or, if no `Complete` deployment is present, the replication controller template in the deployment configuration."